### PR TITLE
adds os update playbook

### PR DESCRIPTION
**Pull request by leefaisonr**

### CODE REVIEW

File: playbooks/os_updates.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
```
Comment (acozine) [inline playbooks/os_updates.yml:2]: Update this to run by default on the `staging` group, with an option to run on `qa` or `production` if you pass a variable at runtime.

File: playbooks/os_updates.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      failed_when: false
+      register: check_python
+
+    - name: Install Python  # this has to be raw in the event python is not installed
+      ansible.builtin.raw: apt -y update && apt install -y python-minimal python-pip
+      when: check_python.rc != 0
+
+  tasks:
+
+    - name: Update apt
```
Comment (acozine) [inline playbooks/os_updates.yml:20]: Combine this task and the one below - `apt-get dist-upgrade` also upgrades packages - it's a twofer!

File: playbooks/os_updates.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
+  serial: 2
+  remote_user: pulsys
+  become: true
+  pre_tasks:
+
+    - name: test for python on new VM
```
Comment (acozine) [inline playbooks/os_updates.yml:8]: We don't need the pre-tasks for an update playbook. Remove these, we will add them in the "bare-VM" playbook (TBD).

File: hosts [hunk 4]
```diff
@@ -49,6 +49,7 @@ lae1.princeton.edu
 lae2.princeton.edu
```
Comment (acozine) [inline hosts:49]: If you rebase this PR, be sure to get rid of these changes - inventory is in `inventory/` since #3258 was merged.

**Review by acozine — COMMENTED** [review]
An excellent addition! Thanks. 

We (@maxkadel @carolyncole @kayiwa @bess @sandbergja and me) reviewed at the Ansible Open House and suggested a few ways to make it even better.

**Comment by kayiwa** [comment]
closes #3324

File: inventory/by_cloud/aws [hunk 5]
```diff
@@ -1 +1,16 @@
-# to be used for AWS resources
\ No newline at end of file
+[production:children]
```
Comment (acozine) [inline inventory/by_cloud/aws:1]: Test to make sure this syntax is additive, and the `production` group will contain these groups AND the ones from `inventory/by_environment`.

**Review by acozine — COMMENTED** [review]
Fantastic! 

One small question about inventory, otherwise this looks great.

**Pull request merged**